my_portfolio.html

```diff
@@ -1,13 +1,63 @@
 ﻿<!DOCTYPE html>
-
 <html lang="en">
+
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
 </head>
+
 <body>
-    <h1>Hello world</h1>
+    <img src="icons/header-illustration-mobile.svg" id="header-img" />
+    <img src="icons/header-desktop.svg" id="header-img-desktop" />
```

Comment:
- [ ] kindly consider adding this image as a background image, not as an `img` because it's a decoration, and decorations are always background images, not `img` tags 😃